MotionMark/tests/simple/resources/tiled-canvas-image.js

```diff
@@ -102,9 +101,9 @@ TiledCanvasImageStage = Utilities.createSubclass(Stage,
             this.context.fillStyle = gradient;
             this.context.fillRect(0, 0, size.width, size.height);
         this.context.restore();
-    },
+    }
 
-    animate: function(timeDelta)
+    animate(timeDelta)
```

Comment:
It will be great if the name includes the units of the time

`timeDelta` -> `timespanInMilliseconds`